app/controllers/api/v1/taxonomies_controller.rb

```diff
@@ -0,0 +1,18 @@
+module Api
+  module V1
+
+    class TaxonomiesController < ApplicationController
+
+      def index
+        # NOTE: We'll need to figure out what's most useful on the front end, this will do for now
+        taxonomies = TaxonomyCode.all.map { |t| "#{t.classification}#{" (#{t.specialization})" if t.specialization}" }
```

Comment:
Should there be a "-" separator between classification and specialization to make the association more explicit?